### PR TITLE
Smith rework one live-view.component

### PR DESCRIPTION
**Pull request by sushshring**

### CODE REVIEW

**Comment by SmithcS** [comment]
Can we close and reopen this? When I push to smith-rework it doesn't update this one.

**Comment by sushshring** [comment]
Should be ready!

File: user-registration-app/src/app/views/schedule-view/schedule-view.component.html [hunk 1]
```diff
@@ -3,123 +3,151 @@
...
-        <div class="col-xs-12 large-screen-hide">
-          <div class="modal-footer">
-            <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
-          </div>
+        <button type="button" href="" class="close small-screen-hide" data-dismiss="modal">&times;</button>
+        <div class="modal-footer large-screen-hide">
+          <button type="button" class="btn btn-flat" data-dismiss="modal">Close</button>
         </div>
       </div>
+      <!--<div class="container-fluid">-->
```
Comment (sushshring) [inline user-registration-app/src/app/views/schedule-view/schedule-view.component.html]: Remove

File: user-registration-app/src/app/views/login/login.component.html [hunk 2]
```diff
@@ -7,8 +7,8 @@
     <h6 class="header-text text-center">LOGIN</h6>
     <!-- Main Form -->
     <div class="row">
-      <form #loginForm="ngForm" (ngSubmit)="loginEmail()" id="login-form" class="col s12">
-        <div class="row">
+      <form #loginForm="ngForm" (ngSubmit)="loginEmail()" id="login-form" class="col-12">
+        <!--<div class="row">-->
```
Comment (sushshring) [inline user-registration-app/src/app/views/login/login.component.html]: remove

File: user-registration-app/src/app/views/rsvp/rsvp.component.html [hunk 3]
```diff
@@ -1,38 +1,33 @@
 <div style="color:#ffffff" *ngIf="rsvpReady(); else rsvpNotOpen">
   <div class="page-margins">
     <div class="page-header text-center">
-      <div style="font-family:'Cornerstone',sans-serif;font-size:75px;"> HackPSU RSVP </div>
+      <div style="font-family:'Cornerstone',sans-serif;font-size:4em;"> HackPSU Information </div>
     </div>
     <!--<ng-template #result>-->
```
Comment (sushshring) [inline user-registration-app/src/app/views/rsvp/rsvp.component.html]: remove

File: user-registration-app/src/app/views/schedule-view/schedule-view.component.ts [hunk 4]
```diff
@@ -70,4 +84,8 @@ export class ScheduleViewComponent implements OnInit {
   getEventDay(event: EventModel) {
     return ScheduleViewComponent.days[new Date(parseInt(event.event_start_time, 10)).getDay()];
   }
+
+  dismissModal() {
```
Comment (sushshring) [inline user-registration-app/src/app/views/schedule-view/schedule-view.component.ts]: not being used remove

**Pull request merged**